JapaneseFilter/sections/specific.txt

```diff
@@ -18,6 +18,11 @@ gigafile.nu###shinkarionz
 gigafile.nu###geniee_overlay_outer
 altema.jp,gorillawiki.jp##.new_koukokuwaku_wrapper
 mainichi.jp###index-remarkable
+touhouyaruosure.com##div[style^="margin:0;padding:5px;"]
+touhouyaruosure.com##a[href^="https://affiliate.suruga-ya.jp"]
+touhouyaruosure.com##a[href^="https://www.dlsite.com/"][target="_blank"]
```

Comment:
`touhouyaruosure.com##a[href^="https://www.dlsite.com/"][target="_blank"]` is it necessary to use `[target="_blank"]`?

Reply:
Can you check `touhouyaruosure.com##.footer ~ div.main a[href^="https://"][rel]` instead of those rules?

Reply:
@zloyden There's a link without [rel] at `http://touhouyaruosure.com/blog-entry-9000.html`

Reply:
@Yuki2718 Ok, I found it. What about these rules?
```
touhouyaruosure.com##.footer ~ div.main a[href^="https://"]:not([href*="twitter.com/"])
touhouyaruosure.com##.main > div[style^="margin:0;padding:5px;"]
```

Reply:
Yup, working fine.

Reply:
Ok, then @stanislav-atr you can exchange your rules to the above ones.